.gitignore

```diff
@@ -0,0 +1,5 @@
+# Ignore the simplecov coverage directory
```

Comment:
thanks for adding this, I didn't know that we could tell git to ignore files before this